plugins/summarize/tests/summarize.cpp

```diff
@@ -158,6 +167,13 @@ TEST(summarize test) {
   CHECK_EQUAL(summarized_slice.at(0, 8), data_view{true});
   CHECK_EQUAL(summarized_slice.at(0, 9), data_view{false});
   CHECK_EQUAL(summarized_slice.at(0, 10), data_view{false});
+  const auto expected_alternating_numbers = list{count{0}, count{1}, count{2}};
+  const auto expected_alternating_numbers_list
+    = list{count{0}, count{1}, count{2}, count{3}, count{4}};
```

Comment:
We need some user-defined literals to express this more conveniently, e.g., `{0_cv, 1_cv, ...}` for "count value".

Reply:
This is definitely not actionable in the scope of this PR. I like the idea overall, but it's also a low priority thing. Mind writing a story?